repo/config/version.go

```diff
@@ -7,13 +7,15 @@ import (
 	"time"
 )
 
+// CurrentVersionNumber and CurrentCommit are normally set via ldflags -X
+
 // CurrentCommit is the current git commit, this is set as a ldflag in the Makefile
 var CurrentCommit string
 
 // CurrentVersionNumber is the current application's version literal
-const CurrentVersionNumber = "0.4.0-dev"
+var CurrentVersionNumber = "0.0.0-dev"
```

Comment:
`CurrentVersionNumber` should stay as it is and not carry commit information -- it's used in too many places where the assumption is a simple vN.N.N string. Maybe just use CurrentCommit?